Hardware/CPU/IntelCPU.cs

```diff
@@ -289,6 +296,7 @@ public IntelCPU(int processorIndex, CPUID[][] cpuid, ISettings settings)
         if (Ring0.Rdmsr(MSR_RAPL_POWER_UNIT, out eax, out edx))
           switch (microarchitecture) {
             case Microarchitecture.Silvermont:
+            case Microarchitecture.Airmont:

```

Comment:
As with nearly any other programming language C# has a _break_ statement. A _case_ statement will execute all blocks of code it can find until the first _break_ which exits the _switch_ (unless you have any other exit like _return_ or an exception). 

This means Silvermont will work the same way as Airmont does.